### PR TITLE
Neculaclaudiu/tema7

### PR DESCRIPTION
**Pull request by Claudiu357**

### CODE REVIEW

File: src/test/java/neculaclaudiu/tests/testNg/BaseTest.java
Comment (95teo) [inline src/test/java/neculaclaudiu/tests/testNg/BaseTest.java:14]: Please add a blank line between methods. Thanks!

File: src/test/java/neculaclaudiu/tests/testNg/CuraHealthcareTesttt.java
Comment (95teo) [inline src/test/java/neculaclaudiu/tests/testNg/CuraHealthcareTesttt.java:19]: Please add a blank line between methods. Thanks!

File: src/test/resources/neculaclaudiu/suite-testng.xml
Comment (95teo) [inline src/test/resources/neculaclaudiu/suite-testng.xml:10]: You can remove lines from 10-15.

File: src/test/resources/neculaclaudiu/suite-testng.xml
Comment (95teo) [inline src/test/resources/neculaclaudiu/suite-testng.xml:6]: Please put the right path to CuraHealthcareTest class.

File: src/test/java/neculaclaudiu/tests/testNg/CuraHealthcareTesttt.java
Comment (95teo) [inline src/test/java/neculaclaudiu/tests/testNg/CuraHealthcareTesttt.java:10]: Please remove the extra 't's from name of the class at the end. Leave only one 't'. CuraHealthcareTesttt -> CuraHealthcareTest. Thanks!

**Review by 95teo — CHANGES_REQUESTED** [review]

**Review by 95teo — APPROVED** [review]

**Comment by 95teo** [comment]
Please don't forget to ask for review from colleagues too. After you have at least 2 approvals (1 from me required) you can merge this PR. Thanks!

**Review by sorinfoca — APPROVED** [review]

**Pull request merged**
